tmpfiles.d/meson.build

```diff
@@ -29,14 +29,15 @@ foreach pair : files
         endif
 endforeach
 
-in_files = [['etc.conf',                      ''],
+in_files = [['20-systemd-shell-extra.conf',   'LINK_SHELL_EXTRA_DROPIN'],
```

Comment:
If you touch this, could you wrap the line after the first `[` ? Then we can reduce change of the lines in future changes. Not a big matter, so feel free to ignore if you do not agree.